app/uk/gov/hmrc/trusts/models/TrustRegistrationModels.scala

```diff
@@ -395,41 +437,54 @@ object NonUKType {
 
 case class PropertyLandType(buildingLandName: Option[String],
                             address: Option[AddressType],
-                            valueFull: Long,
-                            valuePrevious: Long)
+                            valueFull: Option[Long],        // now optional with 5MLD
+                            valuePrevious: Option[Long])    // now optional with 5MLD
 
 object PropertyLandType {
   implicit val propertyLandTypeFormat: Format[PropertyLandType] = Json.format[PropertyLandType]
 }
 
 case class BusinessAssetType(orgName: String,
                              businessDescription: Option[String],
-                             address: AddressType,
-                             businessValue: Long)
+                             address: Option[AddressType],  // now optional with 5MLD
+                             businessValue: Option[Long])   // now optional with 5MLD
 
 object BusinessAssetType {
   implicit val businessAssetTypeFormat: Format[BusinessAssetType] = Json.format[BusinessAssetType]
 }
 
 case class OtherAssetType(description: String,
-                          value: Long)
+                          value: Option[Long])    // now optional with 5MLD
 
 object OtherAssetType {
   implicit val otherAssetTypeFormat: Format[OtherAssetType] = Json.format[OtherAssetType]
 }
 
+// new 5MLD type
+case class NonEEABusinessType(orgName: String,
+                              address: AddressType,
+                              govLawCountry: String,
+                              startDate: LocalDate,
+                              endDate: Option[LocalDate])
+
+object NonEEABusinessType {
+  implicit val format: Format[NonEEABusinessType] = Json.format[NonEEABusinessType]
+}
+
 case class PartnershipType(description: String,
-                           partnershipStart: LocalDate)
+                           partnershipStart: Option[LocalDate])   // now optional with 5MLD
 
 object PartnershipType {
   implicit val partnershipTypeFormat: Format[PartnershipType] = Json.format[PartnershipType]
 }
 
-case class SharesType(numberOfShares: String,
+case class SharesType(numberOfShares: Option[String],   // now optional with 5MLD
```

Comment:
Same with SharesType. We want to mandate these values in the journey and on MDTP.